### ISSUE TITLE
Compared to DANN

### ISSUE THREAD

**Issue by zzpustc** [jindongwang/transferlearning#154]
Hi, 
Thank you for your great work. After reading your paper and code(DAAN), I have a problem.
In DANN code, the objective is L_task + L_domain, but in your code , it seems to be L_task - L_domain.
Can you please tell me the reason? 
Hope to receive your answer.
Best regards.

**Comment by jindongwang** [comment]
Which file?  And which line?

**Comment by zzpustc** [comment]
Thank you for your reply!
In DAAN, on line 150 of train.py, your objective is loss = soft_loss - join_loss. But in DANN the objective is loss = loss_c + loss_d. Meanwhile, when I train DAAN, after about 20 epochs, the loss will be nan.

**Comment by jindongwang** [comment]
@Supe2015

**Comment by Supe2015** [comment]
> Thank you for your reply!
> In DAAN, on line 150 of train.py, your objective is loss = soft_loss - join_loss. But in DANN the objective is loss = loss_c + loss_d. Meanwhile, when I train DAAN, after about 20 epochs, the loss will be nan.

It should be '+', I have fixed it, please git pull and try again.
